nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -116,6 +117,10 @@ BtNavigator::on_configure(const rclcpp_lifecycle::State & /*state*/)
   blackboard->set<bool>("initial_pose_received", false);  // NOLINT
   blackboard->set<int>("number_recoveries", 0);  // NOLINT
 
+  // Odometry smoother object for getting current speed
+  auto node_handle = blackboard->get<rclcpp::Node::SharedPtr>("node");
```

Comment:
Please use the BT Navigator lifecycle node, this node is reserved for the behavior tree nodes